components/scroller/scroller_dynamic.story.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div>
+    <h3>Scroller with dynamic size</h3>
+
+    <button @click="updateItem(0)">
+      Update Element Message
+    </button>
+
+    <dt-scroller
+      :items="dynamicItems"
+      :min-item-size="54"
+      :scroller-height="300"
+      :scroller-width="500"
+      class="scroller"
+      :dynamic="true"
+    >
+      <template #default="{ item }">
+        <div class="avatar">
+          {{ item.id }}
+          <img
+            :key="item.avatar"
+            :src="item.avatar"
+            alt="avatar"
+            class="image"
+          >
+        </div>
+        <div class="text">
+          {{ item.message }}
+        </div>
+      </template>
+    </dt-scroller>
+  </div>
+</template>
```

Comment:
Bit of odd behaviour with this example. When you scroll the scrollbar with the mouse, the scrollbar does not stay in the position where your cursor is. Guessing this has something to do with the dynamic loading.

![2023-03-10 17 08 32](https://user-images.githubusercontent.com/64808812/224456320-477c83f0-cd60-4ba4-8590-ce13f9358c09.gif)

Reply:
Same behaviour in library...  need to investigate it, I recommend to take it later in a different ticket

Reply:
Yeah 100% not a blocker for this PR

Reply:
After looking into it a bit, I think this is an issue that is not possible to fix. Even if you scroll up the chat in slack it has this same issue. The scrollbar itself is going to change if the items inside it are changing, not much we can do about it. 

Don't worry about this one.